tests/hypervolume_tests/test_hssp.py

```diff
@@ -22,39 +21,35 @@ def _compute_hssp_truth_and_approx(test_case: np.ndarray, subset_size: int) -> T
 
 @pytest.mark.parametrize("dim", [2, 3])
 def test_solve_hssp(dim: int) -> None:
-    random.seed(128)
+    rng = np.random.RandomState(128)
 
-    for i in range(8):
-        subset_size = int(random.random() * i) + 1
-        test_case = np.asarray([[random.random() for _ in range(dim)] for _ in range(8)])
+    for i in range(1, 9):
+        subset_size = np.random.randint(1, i + 1)
+        test_case = rng.rand(8, dim)
         truth, approx = _compute_hssp_truth_and_approx(test_case, subset_size)
         assert approx / truth > 0.6321  # 1 - 1/e
 
 
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
 def test_solve_hssp_infinite_loss() -> None:
-    random.seed(128)
+    rng = np.random.RandomState(128)
 
-    subset_size = int(random.random() * 4) + 1
-    test_case = np.asarray([[random.random() for _ in range(2)] for _ in range(8)])
-    test_case = np.vstack([test_case, [float("inf") for _ in range(2)]])
+    subset_size = 4
```

Comment:
In the original test with `random.seed(128)`, this value was `4`, so I fixed it for readability.